### ISSUE TITLE
Incompatibility with architectury.

### ISSUE THREAD

**Issue by Shywizz** [hiisuuii/transparentTooltips#1]
Minecraft version 1.19.2
Architectury 6.5.90
Transparent toolTips 1.0.2

Game crashes on load with these three mods and the required dependencies w/o the sodium mod group

**Comment by hiisuuii** [comment]
I'll check this out as soon as I can. In the mean time, can you provide a crash log? it'll help me figure out what the problem is faster.

**Comment by Shywizz** [comment]
My bad, i kinda posted this in a haste yesterday but here it is 
https://mclo.gs/AxRdB8T
```
Mods:
  [✔] architectury-6.5.90-fabric
  [✔] fabric-api-0.76.0+1.19.2
  [✔] ForgeConfigAPIPort-v4.2.11-1.19.2-Fabric
  [✔] transparent_tooltips-1.0.0+1.19

Params:
  --username  --version 1.19.2 --gameDir /mnt/windows/Applications/Games/PrismLauncher/instances/Survival test/.minecraft --assetsDir /home/shyw/.local/share/PrismLauncher/assets --assetIndex 1.19 --uuid  --accessToken  --userType  --versionType release

Window size: 854 x 480

Launcher: standard

Java Arguments:
[-Xms8192m, -Xmx8192m, -Duser.language=en]


Minecraft process ID: 3366


[08:29:57] [main/INFO]: Loading Minecraft 1.19.2 with Fabric Loader 0.15.6
[08:29:57] [main/INFO]: Loading 57 mods:
	- architectury 6.5.90
	- fabric-api 0.76.0+1.19.2
	   |-- fabric-api-base 0.4.15+8f4e8eb390
	   |-- fabric-api-lookup-api-v1 1.6.14+93d8cb8290
	   |-- fabric-biome-api-v1 9.1.1+16f1e31390
	   |-- fabric-block-api-v1 1.0.2+e415d50e90
	   |-- fabric-blockrenderlayer-v1 1.1.25+cafc6e8e90
	   |-- fabric-client-tags-api-v1 1.0.5+b35fea8390
	   |-- fabric-command-api-v1 1.2.16+f71b366f90
	   |-- fabric-command-api-v2 2.2.1+413cbbc790
	   |-- fabric-commands-v0 0.2.33+df3654b390
	   |-- fabric-containers-v0 0.1.41+df3654b390
	   |-- fabric-content-registries-v0 3.5.2+7c6cd14d90
	   |-- fabric-convention-tags-v1 1.3.0+4bc6e26290
	   |-- fabric-crash-report-info-v1 0.2.8+aeb40ebe90
	   |-- fabric-data-generation-api-v1 5.3.8+413cbbc790
	   |-- fabric-dimensions-v1 2.1.35+0d0f210290
	   |-- fabric-entity-events-v1 1.5.4+9244241690
	   |-- fabric-events-interaction-v0 0.4.34+562bff6e90
	   |-- fabric-events-lifecycle-v0 0.2.36+df3654b390
	   |-- fabric-game-rule-api-v1 1.0.24+b6b6abb490
	   |-- fabric-item-api-v1 1.6.6+b7d1888890
	   |-- fabric-item-groups-v0 0.3.39+9244241690
	   |-- fabric-key-binding-api-v1 1.0.25+5c4fce2890
	   |-- fabric-keybindings-v0 0.2.23+df3654b390
	   |-- fabric-lifecycle-events-v1 2.2.4+1b46dc7890
	   |-- fabric-loot-api-v2 1.1.13+83a8659290
	   |-- fabric-loot-tables-v1 1.1.16+9e7660c690
	   |-- fabric-message-api-v1 5.0.7+93d8cb8290
	   |-- fabric-mining-level-api-v1 2.1.24+33fbc73890
	   |-- fabric-models-v0 0.3.21+c6af733c90
	   |-- fabric-networking-api-v1 1.2.11+10eb22f490
	   |-- fabric-networking-v0 0.3.28+df3654b390
	   |-- fabric-object-builder-api-v1 4.2.2+d8ef690890
	   |-- fabric-particles-v1 1.0.14+4d0d570390
	   |-- fabric-recipe-api-v1 1.0.1+413cbbc790
	   |-- fabric-registry-sync-v0 0.9.32+9244241690
	   |-- fabric-renderer-api-v1 1.2.1+1adbf27790
	   |-- fabric-renderer-indigo 0.8.0+1adbf27790
	   |-- fabric-renderer-registries-v1 3.2.24+df3654b390
	   |-- fabric-rendering-data-attachment-v1 0.3.19+6e0787e690
	   |-- fabric-rendering-fluids-v1 3.0.11+4d0d570390
	   |-- fabric-rendering-v0 1.1.27+df3654b390
	   |-- fabric-rendering-v1 1.12.1+d8ef690890
	   |-- fabric-resource-conditions-api-v1 2.1.2+aae9039d90
	   |-- fabric-resource-loader-v0 0.8.4+edbdcddb90
	   |-- fabric-screen-api-v1 1.0.32+4d0d570390
	   |-- fabric-screen-handler-api-v1 1.3.7+1cc24b1b90
	   |-- fabric-sound-api-v1 1.0.2+c4f28df590
	   |-- fabric-textures-v0 1.0.24+aeb40ebe90
	   |-- fabric-transfer-api-v1 2.1.6+413cbbc790
	   \-- fabric-transitive-access-wideners-v1 1.3.3+08b73de490
	- fabricloader 0.15.6
	   \-- mixinextras 0.3.2
	- java 17
	- minecraft 1.19.2
	- tt 1.0.0+1.19
[08:29:57] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/home/shyw/.local/share/PrismLauncher/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[08:29:58] [main/INFO]: Compatibility level set to JAVA_16
[08:29:58] [main/INFO]: Compatibility level set to JAVA_17
[08:29:58] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.2).
[08:30:01] [main/INFO]: Building optimized datafixer
[08:30:03] [Render thread/WARN]: @ModifyConstant conflict. Skipping tt.client.mixins.json:TransparentToolTips from mod tt->@ModifyConstant::MoreTranslucentTooltipsMod$modifyTooltipBackgroundColor(I)I with priority 1000, already redirected by architectury.mixins.json:client.MixinScreen from mod architectury->@ModifyConstant::modifyTooltipBackgroundColor(I)I with priority 1000
[08:30:03] [Render thread/WARN]: @ModifyConstant conflict. Skipping tt.client.mixins.json:TransparentToolTips from mod tt->@ModifyConstant::MoreTranslucentTooltipsMod$modifyTooltipBackgroundColor(I)I with priority 1000, already redirected by architectury.mixins.json:client.MixinScreen from mod architectury->@ModifyConstant::modifyTooltipBackgroundColor(I)I with priority 1000
[08:30:03] [Render thread/WARN]: @ModifyConstant conflict. Skipping tt.client.mixins.json:TransparentToolTips from mod tt->@ModifyConstant::MoreTranslucentTooltipsMod$modifyTooltipBackgroundColor(I)I with priority 1000, already redirected by architectury.mixins.json:client.MixinScreen from mod architectury->@ModifyConstant::modifyTooltipBackgroundColor(I)I with priority 1000
[08:30:03] [Render thread/WARN]: @ModifyConstant conflict. Skipping tt.client.mixins.json:TransparentToolTips from mod tt->@ModifyConstant::MoreTranslucentTooltipsMod$modifyTooltipBackgroundColor(I)I with priority 1000, already redirected by architectury.mixins.json:client.MixinScreen from mod architectury->@ModifyConstant::modifyTooltipBackgroundColor(I)I with priority 1000
[08:30:03] [Render thread/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_437 failed
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:63) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:472) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.15.6.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_437 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.15.6.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:214) ~[minecraft-1.19.2-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[minecraft-1.19.2-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.6.jar:?]
	... 5 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.15.6.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:214) ~[minecraft-1.19.2-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[minecraft-1.19.2-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.6.jar:?]
	... 5 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Constant modifier method MoreTranslucentTooltipsMod$modifyTooltipBackgroundColor(I)I in tt.client.mixins.json:TransparentToolTips from mod tt failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap client-transparent_tooltips-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:468) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1384) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1062) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:402) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:327) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.15.6.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:214) ~[minecraft-1.19.2-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[minecraft-1.19.2-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.6.jar:?]
	... 5 more
Process exited with code 1.
Log upload triggered at: 22 Feb 2024 08:30:33 +0100
Log uploaded to: https://mclo.gs/AxRdB8T```

**Comment by Shywizz** [comment]
At a glance and with my very little experience just poking around minecraft java modding, it seems one of architectury's classes is doing something before you ??
Again, you might as well discard my guess completely as im really new to this.

**Comment by hiisuuii** [comment]
apparently Architectury also does something with the tooltip background color and we had a conflict. I've fixed this in 1.0.3. Versions other than 1.19.0-1.19.2 do not require this fix. Download: https://modrinth.com/mod/transparent-tooltips/version/dlhQ0siR

**Comment by Shywizz** [comment]
Thank you very much for quickly taking care of the issue.
